meinberlin/apps/plans/assets/PlansMap.jsx

```diff
@@ -260,6 +273,12 @@ class PlansMap extends React.Component {
 
           </form>
         </div>
+        {this.state.showInfoBox &&
+        <div className="map-infobox">
+          <button className="infobox__close" id="close" onClick={this.closeInfoBox.bind(this)}><i className="fa fa-times" /></button>
```

Comment:
I think this should have an aria-label like "close infobox", maybe you can check how we did it for other things closing something (by greping fa-times).

Reply:
the ones that weren't hidden were labelled close but as this could be confusing i added your suggested text also the correct text for the info box, all ready I hope!